profiles/base.yaml

```diff
@@ -170,6 +170,7 @@ exp_timers:
   Thievery: 600
   Attunement: 130
 listen: false
+listen_observe: false
```

Comment:
From the name of this setting, I would expect it to be used everywhere that `DRC.listen?` is called. What are your thoughts?